ethereum_history_api/circuits/lib/src/rlp/decode.nr

```diff
@@ -43,12 +43,16 @@ impl<N> Eq for RlpList<N> {
 }
 
 pub fn extract_payload_len<N>(arr: [u8; N], lenlen: u64) -> u64 {
-    assert(lenlen <= MAX_LEN_IN_BYTES);
+    assert(lenlen <= MAX_LEN_IN_BYTES, "length of length exceeds maximum");
```

Comment:
In all of our new code we start assert messages from uppercase letter. Let's do it here too even if not consistent